<ui version="4.0" >
 <class>NetworkDriveAutoConnectClass</class>
 <widget class="QMainWindow" name="NetworkDriveAutoConnectClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>NetworkDriveAutoConnect</string>
  </property>
  <widget class="QWidget" name="centralWidget" >
   <widget class="QPushButton" name="QBtnExitButton" >
    <property name="geometry" >
     <rect>
      <x>450</x>
      <y>130</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text" >
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar" >
    <property name="geometry" >
     <rect>
      <x>320</x>
      <y>80</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy" >
     <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="baseSize" >
     <size>
      <width>200</width>
      <height>20</height>
     </size>
    </property>
    <property name="layoutDirection" >
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="maximum" >
     <number>1000</number>
    </property>
    <property name="value" >
     <number>0</number>
    </property>
    <property name="alignment" >
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignRight|Qt::AlignTrailing</set>
    </property>
    <property name="textVisible" >
     <bool>false</bool>
    </property>
    <property name="orientation" >
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="textDirection" >
     <enum>QProgressBar::TopToBottom</enum>
    </property>
    <property name="format" >
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5" >
    <property name="geometry" >
     <rect>
      <x>350</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize" >
     <size>
      <width>61</width>
      <height>16</height>
     </size>
    </property>
    <property name="text" >
     <string>リトライ間隔[ms]</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="QLiedRetryTime" >
    <property name="geometry" >
     <rect>
      <x>440</x>
      <y>40</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="acceptDrops" >
     <bool>false</bool>
    </property>
    <property name="text" >
     <string>1000</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QBtnSaveDefault" >
    <property name="geometry" >
     <rect>
      <x>39</x>
      <y>130</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text" >
     <string>起動時設定へ保存</string>
    </property>
   </widget>
   <widget class="QLabel" name="QLblStateInfo" >
    <property name="geometry" >
     <rect>
      <x>99</x>
      <y>170</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
    <property name="alignment" >
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6" >
    <property name="geometry" >
     <rect>
      <x>39</x>
      <y>170</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>状態 ：</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignLeading</set>
    </property>
   </widget>
   <widget class="QPushButton" name="QBtnConnectNow" >
    <property name="geometry" >
     <rect>
      <x>180</x>
      <y>130</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text" >
     <string>すぐに接続</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>20</y>
      <width>255</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize" >
     <size>
      <width>255</width>
      <height>100</height>
     </size>
    </property>
    <layout class="QFormLayout" name="formLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="minimumSize" >
        <size>
         <width>137</width>
         <height>16</height>
        </size>
       </property>
       <property name="text" >
        <string>ローカルネットワークドライブ名</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="QLiedLocalDriveName" />
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="minimumSize" >
        <size>
         <width>130</width>
         <height>16</height>
        </size>
       </property>
       <property name="text" >
        <string>共有ネットワークフォルダパス</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="QLiedConctFolderPass" />
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="minimumSize" >
        <size>
         <width>52</width>
         <height>16</height>
        </size>
       </property>
       <property name="text" >
        <string>ユーザー名</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QLineEdit" name="QLiedLogon_UserName" />
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="label_4" >
       <property name="minimumSize" >
        <size>
         <width>47</width>
         <height>16</height>
        </size>
       </property>
       <property name="text" >
        <string>パスワード</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <widget class="QLineEdit" name="QLiedLogon_Password" />
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="QBtnConnectStart" >
    <property name="geometry" >
     <rect>
      <x>290</x>
      <y>130</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text" >
     <string>自動接続開始</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>599</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar" >
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar" />
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
